src/diffusers/models/attention_dispatch.py

```diff
@@ -1538,18 +1538,65 @@ def _native_attention(
 ) -> torch.Tensor:
     if return_lse:
         raise ValueError("Native attention backend does not support setting `return_lse=True`.")
-    query, key, value = (x.permute(0, 2, 1, 3) for x in (query, key, value))
-    out = torch.nn.functional.scaled_dot_product_attention(
-        query=query,
-        key=key,
-        value=value,
-        attn_mask=attn_mask,
-        dropout_p=dropout_p,
-        is_causal=is_causal,
-        scale=scale,
-        enable_gqa=enable_gqa,
-    )
-    out = out.permute(0, 2, 1, 3)
+    if _parallel_config is None:
```

Comment:
Here, 'supports_context_parallel=True' should be also added to register @sywangyi @sayakpaul

Reply:
thanks, added

Reply:
> @DefTruth that should also fix [#12446 (comment)](https://github.com/huggingface/diffusers/pull/12446#issuecomment-3471590126) right? Could you give this a check?

confirm fixed